### PR TITLE
update(switch.sh): Make rhino-deinst install the latest tagged release rather than the artifacts.

### PR DESCRIPTION
**Pull request by ajstrongdev**

### CODE REVIEW

**Review by cat-master21 — CHANGES_REQUESTED** [review]
This should use the `latest` not 1.0.0 or else I'd have to keep on updating it.

**Comment by ajstrongdev** [comment]
I respectfully disagree, you'd have to keep changing it anyway, however test builds are generated in the artifacts as well, and as such can produce unstable disk images. Our disk image creation should not rely on the development of a utility that is independently developed from the disk image generation.

File: build/switch.sh [hunk 1]
```diff
@@ -26,7 +26,7 @@ mv ~/creation/rhino-config/rhino-config /usr/bin
 # Install rhino-deinst onto system
 mkdir ~/creation/rhino-deinst
 cd ~/creation/rhino-deinst
-wget https://raw.githubusercontent.com/rollingrhinoremix/rhino-deinst/master/rhino-deinst
+wget https://github.com/rollingrhinoremix/rhino-deinst/releases/download/1.0.0/rhino-deinst
```
Comment (cat-master21) [inline build/switch.sh:29]: use "latest/download" so `https://github.com/rollingrhinoremix/rhino-deinst/releases/latest/download/rhino-deinst` this downloads the latest tagged release so 1.0.0 right now.

**Review by cat-master21 — CHANGES_REQUESTED** [review]

File: build/switch.sh (hunk 1 above)
Comment (ajstrongdev) [reply build/switch.sh:29]: I see. Are you able to submit this change or would you prefer if I do it?

Comment (cat-master21) [reply build/switch.sh:29]: Okay then I'll do it then

**Comment by ajstrongdev** [comment]
Once it's completed are you able to merge it and let me know on Matrix/Discord bridge (please ping me), as I need to test the completed disk image.

**Pull request closed without merging**